frontend/src/main/components/UCSBDiningCommonsMenuItem/UCSBDiningCommonsMenuItemTable.js

```diff
@@ -0,0 +1,72 @@
+import OurTable, { ButtonColumn } from "main/components/OurTable";
+import { useBackendMutation } from "main/utils/useBackend";
+import { onDeleteSuccess } from "main/utils/UCSBDiningCommonsMenuItemUtils"
+// import { useNavigate } from "react-router-dom";
+import { hasRole } from "main/utils/currentUser";
+
+
+export function cellToAxiosParamsDelete(cell) {
```

Comment:
Duplicate function definition in UCSBDiningCommonsMenuItemUtils

Reply:
fixed